graph_nets/modules.py

```diff
@@ -118,14 +118,16 @@ def __init__(self,
           use_globals=False,
           received_edges_reducer=reducer)
 
-  def _build(self, graph):
+  def _build(self, graph, edge_kw={}, node_kw={}):
```

Comment:
Having mutable argument defaults is dangerous. 

Instead of using {} as the default please switch to None and then add a statement.

if edge_model_kwargs is None:
  edge_model_kwargs = {}